string robot_side
string source_object_id
geometry_msgs/PoseStamped target_pose
bool flat_hand_mode
---
bool result